app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
```suggestion

```
Czy powinno się takie komentarze zostawiać?

Reply:
co zespół to pewnie inne podejście - ja osobiście zostawiam - ie chce mi się ich pozniej w docsach szukac